src/components/interface/WalletConnect/components/WalletConnectModal/WalletConnectModal.tsx

```diff
@@ -19,7 +20,11 @@ const WalletConnectModal: React.FunctionComponent = () => {
   };
   const handleSelectWallet = (name: WalletName) => {
     handleClose();
-    wallet.connect(name);
+    if (walletOptions[2].name === name) {
```

Comment:
can we do "disconnect" === name? (or whatever the name is for that option). Otherwise, when we add other options we might forget to change the '2' index and get confused

Reply:
True, let's make it more human readable